includes/Modules/Analytics_4.php

```diff
@@ -122,6 +123,44 @@ function( $old_value, $new_value ) {
 				$this->get_method_proxy( 'replicate_analytics_sharing_settings' )
 			);
 		}
+
+		if ( Feature_Flags::enabled( 'gteSupport' ) ) {
```

Comment:
The `add_filter()` call is wrapped in the feature flag check for consistency with the code above it (rather than checking the feature in the hook as per the IB).